config/application.yml.default

```diff
@@ -145,9 +145,6 @@ irs_attempt_api_event_ttl_seconds: 86400
 irs_attempt_api_event_count_default: 1000
 irs_attempt_api_event_count_max: 10000
 irs_attempt_api_payload_size_logging_enabled: true
-kantara_2fa_phone_restricted: false
-kantara_2fa_phone_existing_user_restriction: false
```

Comment:
There's another instance of this `kantara_2fa_phone_existing_user_restriction` under the `test` group which should be removed as well.